opendebates/templates/opendebates/list_ideas.html

```diff
@@ -19,68 +19,77 @@
-      <label for="q">
-        {% blocktrans %}Sort questions by{% endblocktrans %}
-      </label>
-      <select onchange="this.form.submit()"
-              name="sort" class="form-control selectpicker">
-        <option value="">
-          ---
-        </option>
-        <option value="trending" {% if sort == "trending" %}selected{% endif %}>
-          {% blocktrans %}Trending Now{% endblocktrans %}
-        </option>
-        <option value="random" {% if sort == "random" %}selected{% endif %}>
-          {% blocktrans %}Random{% endblocktrans %}
-        </option>
-        <option value="-date" {% if sort == "-date" %}selected{% endif %}>
-          {% blocktrans %}Newest First{% endblocktrans %}
-        </option>
-        <option value="+date" {% if sort == "+date" %}selected{% endif %}>
-          {% blocktrans %}Oldest First{% endblocktrans %}
-        </option>
-        <option value="-votes" {% if sort == "-votes" %}selected{% endif %}>
-          {% blocktrans %}Most Votes{% endblocktrans %}
-        </option>
-        <option value="+votes" {% if sort == "+votes" %}selected{% endif %}>
-          {% blocktrans %}Fewest Votes{% endblocktrans %}
-        </option>
-      </select>
-      <div class="display-count">
-        Displaying <strong>{{ ideas|length }} Questions</strong>
-      </div>
-    </form>
-  </div>
+  {% cache 257 list_ideas_content sort url_name %}
+    <div class="sort-column">
+      <form action="{{ url_name }}" method="GET" class="form-inline">
+        <label for="q">
+          {% blocktrans %}Sort questions by{% endblocktrans %}
+        </label>
+        <select onchange="this.form.submit()"
+                name="sort" class="form-control selectpicker">
+          <option value="">
+            ---
+          </option>
+          <option value="trending" {% if sort == "trending" %}selected{% endif %}>
+            {% blocktrans %}Trending Now{% endblocktrans %}
+          </option>
+          <option value="random" {% if sort == "random" %}selected{% endif %}>
+            {% blocktrans %}Random{% endblocktrans %}
+          </option>
+          <option value="-date" {% if sort == "-date" %}selected{% endif %}>
+            {% blocktrans %}Newest First{% endblocktrans %}
+          </option>
+          <option value="+date" {% if sort == "+date" %}selected{% endif %}>
+            {% blocktrans %}Oldest First{% endblocktrans %}
+          </option>
+          <option value="-votes" {% if sort == "-votes" %}selected{% endif %}>
+            {% blocktrans %}Most Votes{% endblocktrans %}
+          </option>
+          <option value="+votes" {% if sort == "+votes" %}selected{% endif %}>
+            {% blocktrans %}Fewest Votes{% endblocktrans %}
+          </option>
+        </select>
+        <div class="display-count">
+
+            Displaying <strong>{% cache 30 idea_count search_term category.id %}{{ ideas.count }}{% endcache %} Questions</strong>
```

Comment:
thanks for the (in-person) reference. my only comment would be to switch the TTLs around